src/cpp/library/ostream

```diff
@@ -129,7 +129,7 @@ namespace std
 namespace std {
   //Output manipulators
   ostream& endl ( ostream& os ) { return os; }
-  ostream& ends ( ostream& os ) {  }
+  ostream& ends ( ostream& os ) { return os; }
```

Comment:
This function is supposed to alter `os` (by appending a `\0` character). Would it be possible instead of this no-op definition to have just a declaration similar to what you did for streambuf?

Reply:
Done.